.vscode/launch.json

```diff
@@ -1,6 +1,21 @@
 {
     "version": "0.2.0",
     "configurations": [
+        {
+            "name": "Python: debug demisto-sdk",
+            "type": "python",
+            "request": "launch",
+            "program": "${workspaceFolder}/demisto_sdk/__main__.py",
+            "console": "integratedTerminal",
+            "justMyCode": true,
+            "cwd": "${env:CONTENT}", // Predefine the CONTENT variable in your environment first!
+            "env":{"PYTHONPATH": "${workspaceFolder}${pathSeparator}${env:PYTHONPATH}"},
```

Comment:
why do you need the pythonpath? it's automatically pre-defined with the local interpeter.

Reply:
When I try to run this without it, for some reason it runs the demisto-sdk that is globally installed.